terraform/content-store/main.tf

```diff
@@ -67,110 +79,35 @@ resource "aws_ecs_service" "service" {
 #
 
 resource "aws_security_group" "service" {
-  name        = "fargate_${var.service_name}_alb_access"
+  name        = "fargate_${var.service_name}_ingress"
   vpc_id      = data.aws_vpc.vpc.id
-  description = "Allow the internal ALB for the fargate ${var.service_name} service to access the service"
-}
-
-#
-# Internal Load balancer
-#
-
-data "aws_acm_certificate" "internal_elb_cert" {
-  domain   = "*.test.govuk-internal.digital"
-  statuses = ["ISSUED"]
+  description = "Permit internal services to access the ${var.service_name} ECS service"
 }
 
-resource "aws_lb" "internal_alb" {
-  name               = "fargate-${var.service_name}"
-  internal           = "true"
-  load_balancer_type = "application"
-  security_groups    = [aws_security_group.internal_alb_sg.id]
-  subnets            = var.private_subnets
-}
-
-resource "aws_lb_target_group" "internal_lb_tg" {
-  name        = "${var.service_name}-internal"
-  port        = var.container_ingress_port
-  protocol    = "HTTP"
+resource "aws_security_group" "dependencies" {
+  name        = "fargate_${var.service_name}_app"
   vpc_id      = data.aws_vpc.vpc.id
-  target_type = "ip"
-
-  deregistration_delay = 10
+  description = "Allows ingress from ${var.service_name} to its dependencies"
 
-  health_check {
-    path = "/healthcheck"
-  }
-}
-
-resource "aws_lb_listener" "internal_listener" {
-  load_balancer_arn = aws_lb.internal_alb.arn
-  port              = "443"
-  protocol          = "HTTPS"
-  ssl_policy        = "ELBSecurityPolicy-2016-08"
-  certificate_arn   = data.aws_acm_certificate.internal_elb_cert.arn
-
-  default_action {
-    type             = "forward"
-    target_group_arn = aws_lb_target_group.internal_lb_tg.arn
+  egress {
```

Comment:
At some point soon we'll need to switch from inline rules to separate rule resources (because inline will break extensibility when we modularise this). Also not something to worry about in this PR though.